generic/CMakeLists.txt

```diff
@@ -50,7 +50,11 @@ set (srcs
 
   RunEntryPoint.F90
   EntryPointVector.F90
+
+  ESMF_Interfaces.F90
```

Comment:
I recommend we rename this to  `MAPL_ESMF_Interfaces.F90` to avoid confusion.    Minor.

Reply:
File renamed.